File: packages/point/python_std/point_std/money.py
from __future__ import annotations


def formatCentsUsd(cents: int) -> str:
	sign = "-" if cents < 0 else ""
	abs_cents = abs(int(cents))
	dollars, remainder = divmod(abs_cents, 100)
	return f"{sign}${dollars}.{remainder:02d}"

File: packages/point/src/std/money.ts
/** Format integer cents as USD with two decimal places (e.g. 1005 → "$10.05"). */
export function formatCentsUsd(cents: number): string {
	const sign = cents < 0 ? "-" : "";
	const abs = Math.abs(Math.trunc(cents));
	const dollars = Math.floor(abs / 100);
	const remainder = abs % 100;
	return `${sign}$${dollars}.${String(remainder).padStart(2, "0")}`;
}
